lib/index.js

```diff
@@ -3,13 +3,25 @@
  * https://github.com/joyent/node-http-signature
  * Copyright 2012 Joyent, Inc.  All rights reserved.
  *
- * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
 
-const assert = require('assert-plus');
+const env = require('./env');
+let assert;
+let URL;
+if(env.nodejs) {
+  assert = require('assert-plus');
+  URL = require('url').URL;
```

Comment:
elsewhere, I did `global.URL = require(..)` I was not sure of what the impacts of doing `let URL` were in the browser?  What are the effects?

Reply:
Yeah, was a mistake, it should have said `Url`. Fixed.